modules/standard/BigInteger.chpl

```diff
@@ -1089,14 +1089,32 @@ module BigInteger {
 
 
   // Division
-  operator bigint./(const ref a: bigint, const ref b: bigint) {
+  // Documented in (bigint, integral) version
+  operator bigint./(const ref a: bigint, const ref b: bigint): bigint {
     var c = new bigint();
     c.divQ(a, b, round.zero);
 
     return c;
   }
 
-  operator bigint./(const ref a: bigint, b: integral) {
+  /* Divide ``a`` by ``b``, returning the result.
+
+     .. note::
+
+        Because this operator creates a new :record:`bigint` to store the result
+        of the division operation, it may be less efficient than
+        :proc:`bigint./=`, :proc:`bigint.divQ` or the other division methods.
```

Comment:
I wonder if we just want a general note about this in the module docs instead of adding comments to the operators.

Reply:
There is a general note at the start of the module as well, but it isn't linkable (though I can definitely make it linkable)